.dockerignore

```diff
@@ -0,0 +1 @@
+.git
```

Comment:
I still think there are some more ignores to be added, like `*.log`, although being empty for a fresh clone, if you wanted to try it with an existing clone, some of the useless data will be sent to the docker context